functions/src/plugins/common.ts

```diff
@@ -154,9 +155,13 @@ export class CommonTask extends Task {
    * Update the config file when there is a push to master that changes it
    */
   async onPush(context: Context): Promise<void> {
-    let ref = context.payload.ref.split('/');
-    ref = ref[ref.length - 1];
-    if(ref === 'master') {
+    const payload = context.payload as WebhookPayloadPush;
+    const refParts = payload.ref.split('/');
```

Comment:
It will end up being okay because we are only really going to run into `ref/heads/main` and `ref/heads/master` but technically the branchName could have a `/` in it and this would not correctly parse it.

Reply:
Yes, I kept this as is. This logic existed before and I don't think it's worth changing more here